native/src/main/java/io/ballerina/persist/nodegenerator/SyntaxTreeConstants.java

```diff
@@ -0,0 +1,59 @@
+package io.ballerina.persist.nodegenerator;
+
+import io.ballerina.compiler.syntax.tree.AbstractNodeFactory;
+import io.ballerina.compiler.syntax.tree.Token;
+import io.ballerina.compiler.syntax.tree.TypeDescriptorNode;
+import io.ballerina.persist.components.TypeDescriptor;
+
+/**
+ * Class encapsulating all the Syntax tree related constants.
+ *
+ @since 0.1.0
+ */
+public class SyntaxTreeConstants {
+
+    private SyntaxTreeConstants() {
+
+    }
+
+    public static final Token SYNTAX_TREE_SEMICOLON = AbstractNodeFactory.createIdentifierToken(";");
+    public static final Token SYNTAX_TREE_COLON = AbstractNodeFactory.createIdentifierToken(":");
+    public static final Token SYNTAX_TREE_OPEN_BRACE = AbstractNodeFactory.createIdentifierToken("{");
+    public static final Token SYNTAX_TREE_CLOSE_BRACE = AbstractNodeFactory.createIdentifierToken("}");
+
+    public static final Token SYNTAX_TREE_OPEN_CLOSED_BRACE = AbstractNodeFactory.createIdentifierToken("{|");
+    public static final Token SYNTAX_TREE_CLOSE_CLOSED_BRACE = AbstractNodeFactory.createIdentifierToken("|}");
```

Comment:
```suggestion
```

Reply:
Updated in [6d8b4a4](https://github.com/ballerina-platform/persist-tools/pull/15/commits/6d8b4a41b9a0ea1002bb906bb2008380cfa7cdb1)